### DISCUSSION TITLE
Load balancing over multi-site replicated minio

### DISCUSSION THREAD

**Discussion by lucas-dehandschutter** [minio/minio#15653] [Q&A] [answered]
Hello,

Given two minio clusters A and B which are multi-site replicated.

Can we load balance all S3 request fron a client to any node from cluster A and B (with sidekick for example) like we load balance on a single distributed minio cluster?

Thx

**Comment by klauspost** [comment, marked as answer]
You *can*, but there is no read-after-write guarantees across replicated sites.

S3 worked until recently with this, so it might be fine, but some clients may have an expectation of read-after-write.
